core/lib/generators/workarea/install/templates/initializer.rb.erb

```diff
@@ -5,7 +5,11 @@ Workarea.configure do |config|
   config.host = {
     'test' => 'www.example.com',
     'development' => 'localhost:3000',
-    'production' => 'www.<%= app_name.dasherize %>.com' # TODO
+    'production' => if ENV['HEROKU_APP_NAME'].present?
```

Comment:
I'm definitely not happy about adding more code to the initializer, but I honestly couldn't see any other way to do it. Open to suggestions on a better way of setting the site host automatically when we're on Heroku